docs/packages/dara-components/graphs/introduction.mdx

```diff
@@ -126,6 +126,99 @@ def graph_with_meta() -> CausalGraph:
     return cg
 ```
 
+### Graph Legend
+
+By default the graph has a legend which is dependent on the current `editor_mode` selected for that graph. You can gain control over these in two ways:
+
+- You can add to the legends already present with the `additional_legends` property. This accepts an array of legends as can be seen in the example below.
+- Alternatively you can alter the prepopulated legends by passing `default_legends`. This accepts a dict of editor_mode to array of legends.
+
+#### Legend types
+
+There are three kinds of legends which can be added to the graph. Below you will find each kind and the properties they possess.
+
+##### `Legend.Spacer`
+
+Spacers can be used to create a space in between sections of your legend. For example to separate node related legends to edge related ones.
+What it means is that it will not display a symbol. You may still write a label for it.
+
+- `label: string` - Optional label to show in the legend
+
+##### `Legend.Node`
+
+Displays a node symbol, this can be useful to explain what different colors of nodes might mean to an user.
+
+- `label: string` - Optional label to show in the legend
+- `color: string` - Optional color to fill the node symbol in the legend, defaults to theme.blue4
+- `highlight_color: string` - Optional color for the node symbol rim in the legend, defaults to theme.primary
+
+:::tip
+
+When configuring the color properties for legends, the following are supported:
```

Comment:
Not to be resolved in this PR, but we should probably support the same theme color specification for rendering_properties.

Reply:
Agree, would like for colors everywhere to be able to use this, I have made a ticket on Jira to address this :)